data/org.freedesktop.impl.portal.Background.xml

```diff
@@ -102,6 +102,30 @@
       <arg type="b" name="result" direction="out"/>
     </method>
 
+    <!--
+      EnableTimer:
+      @app_id: App id of the application
+      @timer: List of times the app should be started
```

Comment:
I find it kind of weird, that this is a list but named with a singular. While `flags` is named plural. So I guess this should be plural too?